lib/form.coffee

```diff
@@ -113,3 +114,20 @@ exports.ask = (question, callback) ->
 	exports.run [ question ], (error, answers) ->
 		return callback(error) if error?
 		return callback(null, answers[question.name])
+
+###*
+# @summary Fix repeating issue, due to incompatibilities faced with inquirer, when using the 'resin login' command. Use npm promt
+# @function
+# @public
+#
+# @param {Object} question - form question (differs from the inquirer model. see patterns.coffee file for detailed information on the object scheme))
+# @param {Function} callback - callback (err, answer)
+###
+exports.askAuthToken = (question, callback) ->
```

Comment:
The body of the function is correct, but creating a new ask function has the following issues:

- We are sure we won't reuse this function elsewhere, and that it will later be removed.
- The interface it exposes is different to the other `ask` function, therefore introducing confusion.
- The function implies that the form module knows what is going to be asked (an auth token in this case), but the form module should expose an interface to ask questions without knowing what is going to be asked.

Instead, I suggest inlining the body of this function to `loginWithToken` in the patterns module.